cmake/macros/macro_exadg_pickup_exe.cmake

```diff
@@ -28,8 +28,6 @@
 MACRO(EXADG_PICKUP_EXE FILE_NAME TARGET_NAME EXE_NAME)
```

Comment:
It looks like `EXE_NAME` is no longer used?

Reply:
Yes, there is no need anymore. I have to remove it from the interface.